### DISCUSSION TITLE
new MQTT-Subtopics

### DISCUSSION THREAD

**Discussion by digi303** [jomjol/AI-on-the-edge-device#1593] [Q&A]
I recently got new MQTT subtopics, which I can't process properly.
I find the topics interesting:
1) .../main/rate_per_digitalization_round
2) .../main/rate_per_time_unit
3) .../main/rate (these already existed before)

While the first one is logical and provides clear values, number 2 is not quite clear to me. - Especially because no values but only "nan" and "inf" are delivered.

Do I have to specify a time unit in the config. ini (if so, how?) or does it refer to a certain range? 
Then I wonder why I don't get any more values.

And position 3 now unfortunately also only delivers nan and inf. That worked before.
Can anyone say something about this?

Thank you very much.

**Comment by caco3** [comment]
Number 2 (per_time_value) refers to the interval. It is also available in the topic `interval` eg 1 => 1 minute.

![grafik](https://user-images.githubusercontent.com/1783586/207987721-6c6ce56d-931b-4a0b-a276-9ffde4172499.png)

If it showed Nan or Inf it meas that it in general us unable to calculate valid values!

Number 3 is kind of legacy and is per minute if I remember correctly.

**Reply by donnie-darko** [reply]
Hi guys,
I think there is an issue regarding the 'Previous Value' and 'Value' and this is why 'Rate (m³/h)' sensor in Home Assistant always got 'inf' or 'nan' value.
Link to the issue: https://github.com/jomjol/AI-on-the-edge-device/issues/1600

**Reply by caco3** [reply]
This got fixed in https://github.com/jomjol/AI-on-the-edge-device/issues/1578
